reference/5.1/Microsoft.PowerShell.Core/About/about_PSItem.md

```diff
@@ -0,0 +1,230 @@
+---
+description: The automatic variable that contains the current object in the pipeline object.
+Locale: en-US
+ms.date: 12/05/2022
+online version: https://learn.microsoft.com/powershell/module/microsoft.powershell.core/about/about_psitem?view=powershell-7.3&WT.mc_id=ps-gethelp
+schema: 2.0.0
+title: about PSItem
+---
+# about_PSItem
+
+## Short description
+
+The automatic variable that contains the current object in the pipeline object.
+
+## Long description
+
+PowerShell includes the `$PSItem` variable and its alias, `$_`, as
+[automatic variables][01] in scriptblocks that process pipeline objects.
+
+You can use this variable in commands that perform an action on every object or
+on selected objects in a pipeline.
```

Comment:
Is this sentence needed? I am not sure what value it adds and I am not crazy about the structure.

Reply:
Happy to strike this.